pages/account/teach/course/[course-slug]/edit/index.vue

```diff
@@ -0,0 +1,298 @@
+            </bx-btn>
+          </div>
+          <CohortEdit
+            v-else
+            class="teach-course-edit__cohort"
+            :cohort="newCohort"
+            :is-pending="!!newCohort.isPending"
+            @changed="onNewCohortChanged"
+            @cancel-request="() => onCancelCohortEdition(newCohort)"
+            @send-request="() => onSendNewCohort()"
+          >
+            <template #send-text>Create</template>
+          </CohortEdit>
+          <div v-for="cohort in cohorts" :key="cohort.id">
+            <CohortDisplay
+              v-if="!cohort.isEditing"
+              class="teach-course-edit__cohort"
+              :enable-edit-button="true"
+              :cohort="cohort"
+              @edit-request="() => onEditCohortRequest(cohort)"
+            />
+            <CohortEdit
+              v-else
+              class="teach-course-edit__cohort"
+              :cohort="cohort"
+              :is-pending="cohort.isPending!"
+              @cancel-request="() => onCancelCohortEdition(cohort)"
+              @send-request="onSendCohortUpdate"
+            >
+              <template #send-text>Update</template>
+            </CohortEdit>
+          </div>
+        </div>
+      </template>
+    </NuxtLayout>
+  </div>
+</template>
+
+<script setup lang="ts">
+import IconAdd from '@carbon/icons-vue/es/add--alt/16';
+import IconArrowRight from '@carbon/icons-vue/es/arrow--right/16';
+import type { Cohort, Member } from '@/types/cohort';
+import type { Course } from '@/types/course';
+import {
+  getCohortsByCourse,
+  createCohort,
+  updateCohort,
+} from '@/services/cohort';
+import { getCourseIdBySlug } from '@/services/course';
+
+import '@carbon/web-components/es/components/button/index.js';
+import '@carbon/web-components/es/components/loading/index.js';
+import '@carbon/web-components/es/components/skeleton-text/index.js';
+
+definePageMeta({
+  middleware: 'auth',
+});
+
+interface EditableCohort extends Cohort {
+  isEditing?: boolean;
+  isPending?: boolean;
+}
+
+function emptyCohort(): EditableCohort {
+  return {
+    id: '',
+    name: '',
+    members: [],
+    invitations: [],
+    autoEnrollment: false,
+    isEditing: false,
+    isPending: false,
+    enrollmentCode: '',
+  };
+}
+
+const route = useRoute();
+const courseSlug = route.params.courseslug as Course['slug'];
+const courseName = ref('');
+const courseLink = ref('');
+const courseId = ref(undefined as string | undefined);
+
+const newCohort = ref(emptyCohort());
+const cohorts = reactive([] as EditableCohort[]);
+
+const { pending, error, refresh } = useLazyAsyncData(
+  `get-cohorts-by-course-${courseSlug}`,
+  async () => {
+    courseId.value = await getCourseIdBySlug(courseSlug);
+    const { course, cohorts: responseCohorts } = await getCohortsByCourse(
+      courseId.value
+    );
+
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    courseName.value = course.title!;
```

Comment:
```suggestion
    courseName.value = course.title;
```

I think this is not needed.

Reply:
Done [e7eddcc](https://github.com/Qiskit/saiba/pull/336/commits/e7eddcc6e489d2f6e553e9780e0c3ab3aefa7950)